standard_reports/item-usage/latest/src/query.graphql

```diff
@@ -13,12 +13,18 @@ query ItemUsage($storeId: String!, $itemCode: String, $itemName: String) {
         id
         code
         name
-        stats(storeId: $storeId) {
+        stats: stats(storeId: $storeId, amcLookbackMonths: 1) {
```

Comment:
Can't help notice but in the arguments 3 is set for AMC lookback period. Im assuming that this filter typically is used for this filter. That said its read only. Not sure which one to use 1 or 3. Though regardless, IMO should be consistent.

<img width="523" height="68" alt="Image" src="https://github.com/user-attachments/assets/ad98411f-499a-45ba-bfb4-14f9945cb0a9" />